db/db.js

```diff
@@ -10,7 +10,7 @@ const pool = new pg.Pool({
     host: helpers.getEnvVar('PG_HOST'),
     port: helpers.getEnvVar('PG_PORT'),
     user: helpers.getEnvVar('PG_USER'),
-    database: helpers.getEnvVar('PG_USER'),
+    database: helpers.getEnvVar('PG_DATABASE'),
```

Comment:
Can you take a look around to see where else `PG_USER` is being used for both? There is probably stuff in the README and maybe in `setup_postgres.sh` that need to be changed to do this too.

Reply:
Sure can! :) I also just noticed that the env file @mariocimet shared with me doesn't actually include `$PG_CONNECTION_STRING` but it's mentioned in the readme, and I don't know that it's actually being used anywhere

Reply:
It is used after `-d` in `setup_postgresql_local.sh` though and the current env info/template is in 1password so I would also need to update that to be clear we'd need a `$PG_DATABASE` environment variable 🤔

Reply:
there is no `.env.example` file as far as I know is why

Reply:
`$PG_CONNECTION_STRING` was removed by @mariocimet  when he added in `PG_HOST`, `PG_USER`, etc. I guess he forgot to update the README.

Yup, you can update the `.env` in 1Password to include it. It can have extra things that aren't used yet, so you can add it there now so that you don't forget later if you want.

Reply:
My current plan for this is to be converting it back to `PG_USER` before merging because I'm not sure when the best time to update that across things will be, if anyone thinks it should actually be now though I can try to figure that out. I kept forgetting the other day when pushing linting things and it was breaking Travis 😩

Reply:
nvm, it has now been added to the `.travis.yml` file and is in the `.env` on 1password per discussion earlier today :) and after running the build only 4 times it decided to pass with that change 🤨 Better than an automatic break though I guess haha

Reply:
Will update other instances as needed 👍